aspnet/signalr/overview/getting-started/tutorial-high-frequency-realtime-with-signalr.md

```diff
@@ -1,222 +1,276 @@
-
-![The application window](tutorial-high-frequency-realtime-with-signalr/_static/image1.png)
-
-The application created in this tutorial is based on a demo by Damian Edwards. A video containing this demo can be seen [here](https://channel9.msdn.com/Series/Building-Web-Apps-with-ASP-NET-Jump-Start/Building-Web-Apps-with-ASPNET-Jump-Start-08-Real-time-Communication-with-SignalR).
-
-The tutorial will start by demonstrating how to send SignalR messages from each event that fires as the shape is dragged. Each connected client will then update the position of the local version of the shape each time a message is received.
-
-While the application will function using this method, this is not a recommended programming model, since there would be no upper limit to the number of messages getting sent, so the clients and server could get overwhelmed with messages and performance would degrade. The displayed animation on the client would also be disjointed, as the shape would be moved instantly by each method, rather than moving smoothly to each new location. Later sections of the tutorial will demonstrate how to create a timer function that restricts the maximum rate at which messages are sent by either the client or server, and how to move the shape smoothly between locations. The final version of the application created in this tutorial can be downloaded from [Code Gallery](https://code.msdn.microsoft.com/SignalR-20-MoveShape-Demo-6285b83a).
-
-This tutorial contains the following sections:
-
-- [Prerequisites](#prerequisites)
-- [Create the project and add the SignalR and JQuery.UI NuGet package](#createtheproject2013)
-- [Create the base application](#baseapp)
-- [Starting the hub when the application starts](#startup2013)
-- [Add the client loop](#clientloop)
-- [Add the server loop](#serverloop)
-- [Add smooth animation on the client](#animation)
-- [Further Steps](#furthersteps)
-
-<a id="prerequisites"></a>
+In this tutorial, you:
+
+> [!div class="checklist"]
+> * Set up the project
+> * Create the base application
+> * Map to the hub when app starts
+> * Add the client
+> * Run the app
+> * Add the client loop
+> * Add the server loop
+> * Add smooth animation
+
+[!INCLUDE [Consider ASP.NET Core SignalR](~/includes/signalr/signalr-version-disambiguation.md)]
 
 ## Prerequisites
 
-This tutorial requires Visual Studio 2013.
+### Visual Studio 2017
+
+If you don't already have it, [download Visual Studio 2017](https://visualstudio.microsoft.com/downloads/) now.
+
+### ASP.NET and web development workload
+
+When Visual Studio Installer opens, make sure you install the **ASP.NET and web development** workload. It's under **Web & Cloud**.
+
+### NuGet package source API
```

Comment:
Doesn't VS 2017 come already set up this way?, i.e., it shouldn't be necessary to list this as a prereq for VS 2017, I believe you can install the jQuery UI package without doing this.

Reply:
I tried to install the jQuery UI package and received this error:

```
PM> Install-Package jQuery.UI.Combined
Install-Package : The current environment doesn't have a solution open.
At line:1 char:1
+ Install-Package jQuery.UI.Combined
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Install-Package], InvalidOperationException
    + FullyQualifiedErrorId : NuGetNoActiveSolution,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
PM> 
```

Unless I did something wrong, Visual Studio 2017 Community edition does not have the **nuget.org** package source set up. It only has the **Microsoft Visual Studio Offline Packages** installed. I didn't consider the Professional or Enterprise editions.

As you saw, during installation, I selected the **ASP.NET and web development workload** 
option. That option includes **Code tools** > **NuGet package manager**.  The package manager does not have **nuget.org**.

After a fair amount of internet searching, I came to the conclusion that I had to configure the **nuget.org** package. After I did, everything worked great.

If the Professional and Enterprise editions of Visual Studios 2017 do have **nuget.org** already set up, I can move these instructions down to the **Set up the project** section as a note for people who only have the Community edition. 

Please let me know what you'd like me to do. I'm happy to incorporate any changes.

Reply:
Every time I've installed VS, the default source is nuget.org. "no solution open" sounds like a different issue than the source being something other than nuget.org.  None of our tutorials have you set the nuget.org package source, so it should be OK to remove that instruction here.  If we did need it, we would need to update all of our tutorials.